rt/frontend/jaxrs/src/main/java/org/apache/cxf/jaxrs/impl/UriBuilderImpl.java

```diff
@@ -661,6 +662,14 @@ private void setUriParts(URI uri) {
         }
     }
 
+    private boolean queryIsEmpty() {
+        return query == null || query.isEmpty();
+    }
+
+    private boolean sspMatchessPath(final URI uri) {
```

Comment:
`Matchess` is misspelled, `schemeSpecificPartMatchesUriPath` would be probably better name (despite being long).